### PR TITLE
[10.0][T771]- Docker Build of OCB

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
Task [771](https://www.quartile.co/web#id=938&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by YogeshMahera-SerpentCS** [comment]
- Added dependency  `xfonts-75dpi xfonts-base ` for wkhtmltopdf 12.05
- Improved Dockerfile for successfully build

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @yostashiro @TimLai125 Ready for review.

File: Dockerfile [hunk 1]
```diff
@@ -60,18 +62,18 @@ RUN chmod +x /odooboot
 # Get Odoo code
 WORKDIR /opt
 RUN set -x; \
-  git clone --depth 1 https://github.com/oca/ocb.git -b 10.0 odoo \
-  && rm -rf odoo/.git
+  git clone --depth 1 https://github.com/oca/ocb.git -b 10.0 ocb \
+  && rm -rf ocb/.git
```
Comment (yostashiro) [inline Dockerfile:66]: This is not a change we want to implement as it breaks our standard practices of server operation.

File: Dockerfile [hunk 2]
```diff
@@ -60,18 +62,18 @@ RUN chmod +x /odooboot
...
 
 # Change directory owner
-RUN chown -R odoo: /odoo /opt/odoo
+RUN chown -R odoo: ocb /opt/ocb
```
Comment (yostashiro) [inline Dockerfile:69]: Same as above, please revert this change.

File: Dockerfile [hunk 3]
```diff
@@ -60,18 +62,18 @@ RUN chmod +x /odooboot
...
 
 # Install Supervisord.
 # For some reason the boot script does not work (container exits...) when it is directly set to entrypoint.
 RUN apt-get install -y supervisor
 COPY ./supervisord.conf /etc/supervisor/conf.d/
 
-VOLUME ["/odoo", "/usr/share/fonts"]
+VOLUME ["/ocb", "/usr/share/fonts"]
```
Comment (yostashiro) [inline Dockerfile:76]: Revert this change.

**Comment by YogeshMahera-SerpentCS** [comment]
Closing this PR due to commits not squashing.

`serpentcs@serpentcs-pc:~/workspace/openerp/projects/projects_v10/quartile/odoo-docker_new$ git log
commit e76623080db23ed1c15065813d6bc93db5703126 (HEAD -> 10.0)
Author: Yoshi Tashiro <tashiro@quartile.co>
Date:   Mon Apr 13 00:11:49 2020 +0900

    [FIX] Dockerfile: change directory name to odoo
    
    [FIX]Added WKHTMLPDF Version
    
    [FIX] Fix Conflicts 1

commit caf6b345a7634dfee07b6ba74a0dcb4868608b3c
Author: Yoshi Tashiro <tashiro@quartile.co>
Date:   Sun Apr 12 23:50:57 2020 +0900

    [FIX] Dockerfile: revert wkhtmltox version to 0.12.1
    
    With the last commit https://github.com/qrtl/odoo-docker/commit/5a04d8dbaffa93593481370f1465a6e577f78dcb
    docker build fails for some reason.

commit 36283e3a178613d88688a7449d8b662a1abe356e
Author: YogeshMahera-SerpentCS <yogesh.mahera.serpentcs@gmail.com>
Date:   Mon Apr 13 17:54:10 2020 +0800

    [T938][FIX] Improved Docker file code
`

it seems Youshi commit has merged with my commit.
